charts/argo-workflows/templates/controller/workflow-controller-cluster-roles.yaml

```diff
@@ -121,8 +121,18 @@ rules:
   resources:
   - leases
   resourceNames:
+  {{- if .Values.controller.instanceID.enabled }}
+  {{- if .Values.controller.instanceID.useReleaseName }}
```

Comment:
Please indent the template flow control tags. It makes the code easier to read and is consistent with the rest of the file.

Reply:
I've added indent but there is very similar flow control in configMap template:
https://github.com/argoproj/argo-helm/blob/922799081d6037d71971baf3fecf3519c35326e9/charts/argo-workflows/templates/controller/workflow-controller-config-map.yaml#L9-L11
Should I also change it to make files more consistent?

Reply:
That would be nice, please do.

Reply:
Done